src/components/Sum.jsx

```diff
@@ -1 +1,16 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a, b }) {
+  return (
+    <>
+      <p>
+        {`Sum of ${a} and ${b} is ${a + b}`}
+      </p>
+    </>
+  );
```

Comment:
You don't need to use:
1. tag fragment
2. return

And please delete useless indent